qa/tasks/cephfs/mount.py

```diff
@@ -582,7 +582,7 @@ def remount(self, **kwargs):
 
         self.update_attrs(**kwargs)
 
-        retval = self.mount(mntopts=mntopts, check_status=check_status)
+        retval = self.mount(mntopts=mntopts, check_status=check_status, **kwargs)
```

Comment:
Passing `kwargs` is redundant because each k-v pair in kwargs are set object attributes by `update_attrs()`

Reply:
See commit message in `qa: pass kwargs to mount from remount`.